### ISSUE TITLE
indent rule uses ClassBody node to determine indent level, causing false positives

### ISSUE THREAD

**Issue by zpao** [eslint/eslint#5764]
**What version of ESLint are you using?**
master (currently 2.5.3)

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
es6, indent: 2, 2

**What did you do? Please include the actual source code causing the issue.**

```js
class Foo
  extends Bar {
  fn() {}
}
```

**What did you expect to happen?**
No indentation rule vioations

**What actually happened? Please include the actual, raw output from ESLint.**
Expected indentation of 4 space characters but found 2.

* * *

Basically (IMO) the indent rule should be looking at the ClassDeclaration node to determine the indent level but it's using the ClassBody node instead.

**Comment by kaicataldo** [comment]
Able to reproduce with both v2.5.3 and v2.6.0
